brocaar/chirpstack-api
# Include official python url in main README. Set classifiers and long …

### Contribution description
This commit sets documentation to be seen in PyPi site. README file have been moved to package directory. In addition to this, it includes a set of classifiers that fit with the library. 

### Testing procedure

- Install twine
- Create source distribution `python setup.py sdist`
- Validate that long description README file comply with PyPi. This will prevent raw source instead of rendering. `twine check dist/*`
